script/collect.js

```diff
@@ -118,11 +118,14 @@ async function main () {
   const oldNpmCount = old.find(release => release.npm_package_name === 'electron').length
```

Comment:
Side note, this code doesn't work either.  It should be `.filter` instead of `.find` if we are using `.length`, unless I'm missing something here.

cc @zeke